alicloud/resource_alicloud_polardb_account.go

```diff
@@ -0,0 +1,203 @@
+		}
+		return WrapError(err)
+	}
+
+	parts, err := ParseResourceId(d.Id(), 2)
+	if err != nil {
+		return WrapError(err)
+	}
+
+	d.Set("db_cluster_id", parts[0])
+	d.Set("account_name", object.AccountName)
+	d.Set("account_type", object.AccountType)
+	d.Set("account_description", object.AccountDescription)
+
+	return nil
+}
+
+func resourceAlicloudPolarDBAccountUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	polarDBService := PolarDBService{client}
+	d.Partial(true)
+	parts := strings.Split(d.Id(), COLON_SEPARATED)
+	instanceId := parts[0]
+	accountName := parts[1]
+
+	if d.HasChange("account_description") {
+		if err := polarDBService.WaitForPolarDBAccount(d.Id(), Available, DefaultTimeoutMedium); err != nil {
+			return WrapError(err)
+		}
+		request := polardb.CreateModifyAccountDescriptionRequest()
+		request.RegionId = client.RegionId
+		request.DBClusterId = instanceId
+		request.AccountName = accountName
+		request.AccountDescription = d.Get("account_description").(string)
+
+		raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+			return polarDBClient.ModifyAccountDescription(request)
+		})
+		if err != nil {
+			return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		d.SetPartial("account_description")
+	}
+
+	if d.HasChange("account_password") {
+		if err := polarDBService.WaitForPolarDBAccount(d.Id(), Available, DefaultTimeoutMedium); err != nil {
+			return WrapError(err)
+		}
+		request := polardb.CreateModifyAccountPasswordRequest()
+		request.RegionId = client.RegionId
+		request.DBClusterId = instanceId
+		request.AccountName = accountName
+		request.NewAccountPassword = d.Get("account_password").(string)
+
+		raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+			return polarDBClient.ModifyAccountPassword(request)
+		})
+		if err != nil {
+			return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		d.SetPartial("account_password")
+	}
+
+	d.Partial(false)
+	return resourceAlicloudPolarDBAccountRead(d, meta)
+}
+
+func resourceAlicloudPolarDBAccountDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	polarDBService := PolarDBService{client}
+	parts, err := ParseResourceId(d.Id(), 2)
+	if err != nil {
+		return WrapError(err)
+	}
+	request := polardb.CreateDeleteAccountRequest()
+	request.RegionId = client.RegionId
+	request.DBClusterId = parts[0]
+	request.AccountName = parts[1]
+
+	raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+		return polarDBClient.DeleteAccount(request)
+	})
+	if err != nil && !IsExceptedError(err, InvalidAccountNameNotFound) {
```

Comment:
I think InvalidAccountNameNotFound should return nil

Reply:
> I think InvalidAccountNameNotFound should return nil

done